kind/README.adoc

```diff
@@ -0,0 +1,204 @@
+= How to create a Kubernetes cluster locally
+:icons: font
+:revdate: {docdate}
+:toc: left
+:toclevels: 2
+
+== Introduction
+
+The goal of this project is to simplify our life when we use kubernetes
+on a laptop. It currently supports the following features: 
+
+* Create/delete a K8s cluster using the latest cluster version, or the one you specify, 
+* Launch a local container registry exposed on port `localhost:5000`, 
+* Set up an Ingress
+controller to route the traffic. 
+
+The bash script has been developed supporting the installation of a docker registry according to the following options: 
```

Comment:
`installation of a docker registry according` -> `installation of a container images registry according` to stay consistent with what you said before

Reply:
done